### PR TITLE
[REF] test_pylint: Use valid_odoo_versions from version environment variable

### PR DESCRIPTION
**Pull request by moylop260**
Second part of https://github.com/OCA/pylint-odoo/pull/77

Full fix https://github.com/OCA/maintainer-quality-tools/issues/366

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please include a module with invalid version and increase expected errors to see if it works

**Comment by moylop260** [comment]
Done!

Currently we have that case in the [7.0 build](https://travis-ci.org/OCA/maintainer-quality-tools/jobs/167648707#L463)
 - `broken_deprecated/__openerp__.py:2: [C8106(manifest-version-format), ] Wrong Version Format "8.0.0.1.0.0" in manifest file. Regex to match: "(7.0).\d+.\d+.\d+"`

I just increase the PYLINT_EXPECTED_ERRORS for that build

**Comment by moylop260** [comment]
https://github.com/OCA/pylint-odoo/pull/79 is not required for this one

**Comment by moylop260** [comment]
Travis green @lasley @jjscarafia @sbidoul @dreispt your feedback is welcome

File: travis/test_pylint [hunk 1]
```diff
@@ -146,6 +147,10 @@ if version:
         extra_params_cmd.extend([
             '--extra-params', extra_param
         ])
+    is_version_number = re.match('\d+.\d+', version)
```
Comment (lasley) [inline travis/test_pylint]: `r'\d+\.\d+'`

File: travis/test_pylint [hunk 2]
```diff
@@ -147,7 +147,7 @@ if version:
         extra_params_cmd.extend([
             '--extra-params', extra_param
         ])
-    is_version_number = re.match('\d+.\d+', version)
+    is_version_number = re.match(r'\d+.\d+', version)
```
Comment (lasley) [inline travis/test_pylint]: There needs to also be a `\` in front of the `.`, which is a Regex wildcard

**Comment by moylop260** [comment]
fixed

**Comment by lasley** [comment]
There needs to also be a \ in front of the ., which is a Regex wildcard

**Pull request merged**